extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -415,6 +417,49 @@ export class AmpStoryEmbeddedComponent {
     return parseUrlDeprecated(elUrl).href;
   }
 
+  /**
+   * Animates into expanded view.
+   * @param {!Element} target
+   * @private
+   */
+  animateExpanded_(target) {
+    const state = {};
+    this.resources_.measureMutateElement(target,
+        /** measure */
+        () => {
+          const targetRect = target./*OK*/getBoundingClientRect();
+          const storyPage =
+            this.storyEl_.querySelector('amp-story-page[active]');
+          const pageRect = storyPage./*OK*/getBoundingClientRect();
+
+          const centeredTop = pageRect.height / 2 - targetRect.height / 2;
+          const centeredLeft = pageRect.width / 2 - targetRect.width / 2;
+
+          // Only account for offset from target to page borders. Since in
+          // desktop mode page is not at the borders of viewport.
+          const leftOffset = targetRect.left - pageRect.left;
+          const topOffset = targetRect.top - pageRect.top;
+
+          state.translateY = centeredTop - topOffset;
+          state.translateX = leftOffset - centeredLeft;
+        },
+        /** mutate */
+        () => {
+          target.classList.add('i-amphtml-animate-expand-in');
+          setImportantStyles(dev().assertElement(target),
+              {
+                'transform': 'translate3d(' + state.translateX + 'px, ' +
```

Comment:
Nit: you shouldn't need the quotes around `'transform'`. Also, can you please use an ES6 template string for the `translate3d`?

Reply:
Done.